From 1cd2ba2bebcd49d92e9b8b9542b7f9d04b925fea Mon Sep 17 00:00:00 2001
From: Lars Greiss <kufikugel@googlemail.com>
Date: Sat, 19 Apr 2014 15:45:17 +0200
Subject: [PATCH 033/296] Frameworks: apple can do...we can do as well

incredible apple engineers speeded up the overall UI and reduced per default slightly
the animation scales. Well we do as well and reduce it from 1 to 0.5...smooth, looks well
and does not break anything
---
 services/core/java/com/android/server/wm/WindowManagerService.java | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/services/core/java/com/android/server/wm/WindowManagerService.java b/services/core/java/com/android/server/wm/WindowManagerService.java
index cfeb98bc098..fb968461b2f 100644
--- a/services/core/java/com/android/server/wm/WindowManagerService.java
+++ b/services/core/java/com/android/server/wm/WindowManagerService.java
@@ -713,9 +713,9 @@ public class WindowManagerService extends IWindowManager.Stub
     PowerManager mPowerManager;
     PowerManagerInternal mPowerManagerInternal;
 
-    private float mWindowAnimationScaleSetting = 1.0f;
-    private float mTransitionAnimationScaleSetting = 1.0f;
-    private float mAnimatorDurationScaleSetting = 1.0f;
+    private float mWindowAnimationScaleSetting = 0.5f;
+    private float mTransitionAnimationScaleSetting = 0.5f;
+    private float mAnimatorDurationScaleSetting = 0.5f;
     private boolean mAnimationsDisabled = false;
 
     final InputManagerService mInputManager;
-- 
2.11.0

